### PR TITLE
remove unused parameters from `HLTPixelIsolTrackL1TFilter` plugin

### PR DESCRIPTION
**Pull request by missirol**
#### PR description:

This PR removes the unused parameters `L1GTSeedLabel` and `MinDeltaPtL1Jet` from the plugin `HLTPixelIsolTrackL1TFilter`.

Minor technical improvements are also applied to the latter plugin.

a49ed25c86ac20c19f72032343e59c7b8a90e6f9 is an unrelated minor change to address https://github.com/cms-sw/cmssw/pull/41058#discussion_r1139306442.

Merely technical. No changes expected.

#### PR validation:

None.

#### If this PR is a backport, please specify the original PR and why you need to backport that PR. If this PR will be backported, please specify to which release cycle the backport is meant for:

N/A

### CODE REVIEW

**Comment by missirol** [comment]
please test

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py [hunk 1]
```diff
@@ -210,13 +210,20 @@ def customiseForOffline(process):
 
     return process
 
-def customizeForRecoPixelVertexing(process):
+def customizeHLTfor41058(process):
```
Comment (missirol) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py:213]: To address https://github.com/cms-sw/cmssw/pull/41058#discussion_r1139306442 (just fyi, @mmusich).

**Comment by missirol** [comment]
please test

**Comment by missirol** [comment]
please test

Realised that one more parameter was unused, so I removed it (PR description updated). Hopefully, this is really the last time I touch this PR.

**Comment by missirol** [comment]
please abort

**Comment by missirol** [comment]
please test

**Comment by perrotta** [comment]
+1

**Comment by perrotta** [comment]
merge

**Pull request merged**